op-service/txmgr/metrics/tx_metrics.go

```diff
@@ -26,9 +49,72 @@ func MakeTxMetrics(ns string, factory metrics.Factory) TxMetrics {
 			Help:      "L1 gas fee for transactions in GWEI",
 			Subsystem: "txmgr",
 		}),
+		currentNonce: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Name:      "current_nonce",
+			Help:      "",
+			Subsystem: "txmgr",
+		}),
+		txConfirmed: factory.NewCounterVec(prometheus.CounterOpts{
+			Namespace: ns,
+			Name:      "tx_confirmed_count",
+			Help:      "",
+			Subsystem: "txmgr",
+		}, []string{"status"}),
+		txPublished: factory.NewCounter(prometheus.CounterOpts{
+			Namespace: ns,
+			Name:      "tx_published_count",
+			Help:      "",
+			Subsystem: "txmgr",
+		}),
+		txPublishError: factory.NewCounterVec(prometheus.CounterOpts{
+			Namespace: ns,
+			Name:      "tx_publish_error_count",
+			Help:      "",
+			Subsystem: "txmgr",
+		}, []string{"error"}),
+		lastPublishTime: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Name:      "last_publish_time_unix_secs",
+			Help:      "",
+			Subsystem: "txmgr",
+		}),
+		lastConfirmTime: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Name:      "last_confirm_time_unix_secs",
+			Help:      "",
+			Subsystem: "txmgr",
+		}),
+		rpcError: factory.NewCounter(prometheus.CounterOpts{
```

Comment:
Wondering if we should make this a `CounterVec` with one dimension. Many different rpc requests are currently all just bumping this counter, not sure how helpful this will be.

Reply:
It's more information than we had, but definitely doesn't provide much actionable information. I have some ideas for shared RPC metrics that I think will work better (but is future work)